resources/views/peptideSummary.html

```diff
@@ -311,16 +311,28 @@
 
     function customDateRange() {
         $('#fom-loading').show();
-        if (document.getElementById("ps-start-date").value) {
-            startDate = formatDate(document.getElementById("ps-start-date").value);
+        const startDateVal = document.getElementById('ps-start-date').value;
+        const endDateVal = document.getElementById('ps-end-date').value;
+
+        if (!startDateVal || !endDateVal) {
+            alert('Please select both start and end dates.');
+            $('#fom-loading').hide();
+            return;
+        }
+
+        if (startDateVal) {
```

Comment:
Are these `if` statements doing anything? We've already checked them both on line 317.